activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Filter `I18n.load_path`s passed to the file watcher to only those under the `Rails.root` paths.
```

Comment:
This captures the what but not the why. You might want to mention that this is a speed optimization. We should also post a rudimentary benchmark before merging.

Reply:
I've included a benchmark in the description. It compares main to this PR for `i18n.initalize_i18n`. It examines 17 different levels of "local" vs "gem" based locale files.

Reply:
When there are *no* gem translation files the difference seems nearly within the margin of error. But as the number of gem translations grows the performance does grow a fair bit.